packages/lodestar/test/e2e/doppelganger/doppelganger.test.ts

```diff
@@ -0,0 +1,301 @@
+import {fromHexString} from "@chainsafe/ssz";
+import {generateAttestationData} from "@chainsafe/lodestar-beacon-state-transition/test/utils/attestation";
+import {SLOTS_PER_EPOCH} from "@chainsafe/lodestar-params";
+
+chai.use(chaiAsPromised);
+
+/* eslint-disable @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment */
+describe("doppelganger / doppelganger test", function () {
+  const afterEachCallbacks: (() => Promise<unknown> | void)[] = [];
+  afterEach(async () => {
+    while (afterEachCallbacks.length > 0) {
+      const callback = afterEachCallbacks.pop();
+      if (callback) await callback();
+    }
+  });
+
+  const validatorCount = 1;
+  const SECONDS_PER_SLOT = 2;
+  const genesisSlotsDelay = 5;
+  const beaconParams: Pick<IChainConfig, "SECONDS_PER_SLOT"> = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    SECONDS_PER_SLOT: SECONDS_PER_SLOT,
+  };
+
+  const timeout = (SLOTS_PER_EPOCH + genesisSlotsDelay) * beaconParams.SECONDS_PER_SLOT * 1000;
+
+  it("should shut down validator if same key is active", async function () {
+    this.timeout("10 min");
+
+    const enableDoppelganger = true;
+    const doppelgangerEpochsToCheck = 2;
+    const testLoggerOpts: TestLoggerOpts = {logLevel: LogLevel.info};
+    const loggerNodeA = testLogger("Node-A", testLoggerOpts);
+    const loggerNodeB = testLogger("Node-B", testLoggerOpts);
+
+    const bn = await getDevBeaconNode({
+      params: beaconParams,
+      options: {sync: {isSingleNode: true}},
+      validatorCount,
+      logger: loggerNodeA,
+    });
+    afterEachCallbacks.push(() => bn.close());
+
+    const {validators: validatorsWithDoppelganger} = await getAndInitDevValidators({
+      node: bn,
+      validatorsPerClient: validatorCount,
+      validatorClientCount: 1,
+      startIndex: 0,
+      useRestApi: false,
+      testLoggerOpts,
+      enableDoppelganger,
+      doppelgangerEpochsToCheck,
+    });
+    afterEachCallbacks.push(() => Promise.all(validatorsWithDoppelganger.map((v) => v.stop())));
+
+    const bn2 = await getDevBeaconNode({
+      params: beaconParams,
+      options: {api: {rest: {enabled: false}}},
+      validatorCount,
+      genesisTime: bn.chain.getHeadState().genesisTime,
+      logger: loggerNodeB,
+    });
+    afterEachCallbacks.push(() => bn2.close());
+
+    const {validators} = await getAndInitDevValidators({
+      node: bn2,
+      validatorsPerClient: validatorCount,
+      validatorClientCount: 1,
+      startIndex: 0,
+      useRestApi: false,
+      testLoggerOpts,
+    });
+    afterEachCallbacks.push(() => Promise.all(validators.map((v) => v.stop())));
+
+    await connect(bn2.network as Network, bn.network.peerId, bn.network.localMultiaddrs);
+
+    await Promise.all([...validatorsWithDoppelganger, ...validators].map((validator) => validator.start()));
+
+    expect(validators[0].getStatus()).to.be.equals(
```

Comment:
To be more correct english I guess `to.equal` or `to.be.equal` would make more sense than `to.be.equals` right?